cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectAssemblyOperator.java

```diff
@@ -43,18 +45,20 @@
      * @param configMapOperations For operating on ConfigMaps
      * @param deploymentOperations For operating on Deployments
      * @param serviceOperations For operating on Services
+     * @param secretOperations For operating on Secrets
      */
     public KafkaConnectAssemblyOperator(Vertx vertx, boolean isOpenShift,
                                         ConfigMapOperator configMapOperations,
                                         DeploymentOperator deploymentOperations,
-                                        ServiceOperator serviceOperations) {
-        super(vertx, isOpenShift, AssemblyType.CONNECT, configMapOperations);
+                                        ServiceOperator serviceOperations,
+                                        SecretOperator secretOperations) {
+        super(vertx, isOpenShift, AssemblyType.CONNECT, configMapOperations, secretOperations);
         this.serviceOperations = serviceOperations;
         this.deploymentOperations = deploymentOperations;
     }
 
     @Override
-    protected void createOrUpdate(Reconciliation reconciliation, ConfigMap assemblyCm, Handler<AsyncResult<Void>> handler) {
+    protected void createOrUpdate(Reconciliation reconciliation, ConfigMap assemblyCm, List<Secret> assemblySecrets, Handler<AsyncResult<Void>> handler) {
```

Comment:
This seems a bit unfortunate design that this signature is shared across all AssemblyOperators. But probably nothing to change within this PR.

TBH, from the previous discussions I understood that this will be the place where the `Description` class will be used to make it possible to wrap the different requirements for each `*Cluster` class.